core/src/main/scala/spinal/core/Bits.scala

```diff
@@ -22,127 +22,157 @@ package spinal.core
 
 import spinal.core.internals._
 
-/**
-  * Bits factory used for instance by the IODirection to create a in/out Bits
+/** Bits factory used for instance by the IODirection to create a in/out Bits
   */
 trait BitsFactory {
+
   /** Create a new Bits */
   def Bits(u: Unit = null) = new Bits()
+
   /** Create a new Bits of a given width */
   def Bits(width: BitCount): Bits = Bits().setWidth(width.value)
 }
 
-
-/**
-  * The Bits type corresponds to a vector of bits that does not convey any arithmetic meaning.
+/** The Bits type corresponds to a vector of bits that does not convey any arithmetic meaning.
   *
-  * @example {{{
-  *     val myBits1 = Bits(32 bits)
-  *     val myBits2 = B(25, 8 bits)
-  *     val myBits3 = B"8'xFF"
-  *     val myBits4 = B"1001_0011
-  * }}}
+  * @example
+  *   {{{ val myBits1 = Bits(32 bits) val myBits2 = B(25, 8 bits) val myBits3 = B"8'xFF" val myBits4 = B"1001_0011 }}}
   *
-  * @see  [[http://spinalhdl.github.io/SpinalDoc/spinal/core/types/Bits Bits Documentation]]
+  * @see
+  *   [[http://spinalhdl.github.io/SpinalDoc/spinal/core/types/Bits Bits Documentation]]
   */
-class Bits extends BitVector with DataPrimitives[Bits] with BaseTypePrimitives[Bits] with BitwiseOp[Bits]{
+class Bits extends BitVector with DataPrimitives[Bits] with BaseTypePrimitives[Bits] with BitwiseOp[Bits] {
 
-  override def getTypeObject  = TypeBits
+  override def getTypeObject = TypeBits
 
   override def opName: String = "Bits"
 
   override type T = Bits
 
   private[spinal] override def _data: Bits = this
 
-  /**
-    * Concatenation between two Bits
-    * @example{{{ val myBits2 = bits1 ## bits2 }}}
-    * @param right a Bits to append
-    * @return a new Bits of width (w(this) + w(right))
+  /** Concatenation between two Bits
+    * @example{{{
+    *   val myBits2 = bits1 ## bits2 }}}
+    * @param right
+    *   a Bits to append
+    * @return
+    *   a new Bits of width (w(this) + w(right))
     */
   def ##(right: Bits): Bits = wrapBinaryOperator(right, new Operator.Bits.Cat)
 
   /* Operator defined in BitwiseOp */
-  override def |(right: Bits): Bits = wrapBinaryOperator(right, new Operator.Bits.Or)
-  override def &(right: Bits): Bits = wrapBinaryOperator(right, new Operator.Bits.And)
-  override def ^(right: Bits): Bits = wrapBinaryOperator(right, new Operator.Bits.Xor)
+  override def |(right: Bits): Bits =
+    wrapBinaryOperator(right, new Operator.Bits.Or)
```

Comment:
Hmm any idea why that one was put on two lines ? while the def ## just above stayed on a single line ?
(i love single lines <3)

Reply:
Fixed. I was using git rebase and the code format tool is not picking up the latest configuration.